kubetest2/kubetest2-gce/deployer/build_test.go

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package deployer
+
+import (
+	"os"
+	"testing"
+)
+
+func TestSetRepoPathIfNotSet(t *testing.T) {
+	cases := []struct {
+		name string
+
+		deployerFromFlags deployer
```

Comment:
nit: deployerFromFlags doesn't sound acccurate maybe something simpler?

Reply:
Changed to `initialDeployer`.